### PR TITLE
Clarify how minimal bound for implicit update is defined

### PR DESCRIPTION
**Pull request by domenkozar**

### CODE REVIEW

**Review by gromakovsky — APPROVED** [review]
Ok, except that I don't know about formatting rules in this repository (text seems to have a limit on its width which you likely violated).

**Review by georgeee — APPROVED** [review]

File: _docs/cardano/2017-01-16-updating.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ The fact that stakeholders are responsible for system updates does not restrict
 us to a system where every single update requires a signature from the majority
 of stake. We can introduce the concept of an **implicit agreement**.
 
-An update has to have at least `T%` of the stake signatures to be published on
+An update has to have at least minimal bound (`updateProposalThd` in configuration.yaml) of the stake signatures to be published on
```
Comment (denisshevchenko) [inline _docs/cardano/2017-01-16-updating.md]: > `updateProposalThd` in configuration.yaml

Please add a link to this configuration file: 

```
[configuration.yaml](https://github.com/input-output-hk/cardano-sl/blob/e26ad11397c87ce8b00f2a26d5e237f54d6ea90a/lib/configuration.yaml).
```

**Review by denisshevchenko — CHANGES_REQUESTED** [review]

File: _docs/cardano/2017-01-16-updating.md (hunk 1 above)
Comment (denisshevchenko) [reply _docs/cardano/2017-01-16-updating.md]: I'm sure it would be more convenient for the reader to know where this configuration file lives. :-)

**Comment by domenkozar** [comment]
done

**Review by denisshevchenko — APPROVED** [review]

**Pull request merged**